int32 workerID
---
bool isActive
bool isComplete
int32 workerID
string type
geometry_msgs/Pose[] zoneCorners
int32 zoneID
geometry_msgs/Pose startingPoint
